README.md

```diff
@@ -1,19 +1,9 @@
 # build-your-own-internet
 
 ## How to approach this repo
-The intention behind this project is discovery and understanding. The idea is to
-explore how the internet is put together and use common industry tools to see
-and understand all the communications necessary to make the services and apps we
-use work.
-
-We decided to take notes that create a reproducable experience. We document our
-questions, discovery, and our processes so other people who may want to take a
-similar journey can have a path to follow. We have broken up our exploration
-into chapters. Each chapter has a README that documents our exploration and
-**the final state** for that chapter of whatever files we create or modify in
-our exploration. There is, at least as of yet, very little actual software to
-run from this repo. Instead, the intent is to guide the experience and the
-learning process.
+The intention behind this project is discovery and understanding. The idea is to explore how the internet is put together and use common industry tools to see and understand all the communications necessary to make the services and apps we use work.
+
+We decided to take notes that create a reproducible experience. We document our questions, discovery, and our processes so other people who may want to take a similar journey can have a path to follow. We have broken up our exploration into chapters. Each chapter has a README that documents our exploration and **the final state** for that chapter of whatever files we create or modify in our exploration. There is, at least as of yet, very little actual software to run from this repo. Instead, the intent is to guide the experience and the learning process.
```

Comment:
> **the final state** for that chapter

is this still going to be true? Aren't we planning on setting up chapter 4 to be a beginning state?